docs/tutorials_examples/tutorial/vagrant-setup.rst

```diff
@@ -36,6 +36,20 @@ These two IP addresses are used throughout the :ref:`MongoDB tutorial<tutorial-m
    This will also require you to start the node services manually.
    For more information, see :ref:`authentication`.
 
+Installing Vagrant and VirtualBox
+=================================
+
+Follow the instructions specific to your platform that are provided in the `Vagrant`_ and `VirtualBox`_ documentation.
+
+Alternatively, on OS X you can use ``brew-cask`` to install the packages using the `Homebrew`_ package manager.
+For example:
+
+.. prompt:: bash $
```

Comment:
:grey_question: The rest of this tutorial uses `alice@mercury` for the prompt. Do we have policy on when docs are supposed to have which prompt? Totally your call which to use here, just thought I'd point out that it was different.